src/cmd/dev.go

```diff
@@ -249,19 +255,21 @@ var devLintCmd = &cobra.Command{
 	Aliases: []string{"l"},
 	Short:   lang.CmdDevLintShort,
 	Long:    lang.CmdDevLintLong,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		pkgConfig.CreateOpts.BaseDir = common.SetBaseDirectory(args)
 		v := common.GetViper()
 		pkgConfig.CreateOpts.SetVariables = helpers.TransformAndMergeMap(
 			v.GetStringMapString(common.VPkgCreateSet), pkgConfig.CreateOpts.SetVariables, strings.ToUpper)
 		validator, err := lint.Validate(cmd.Context(), pkgConfig.CreateOpts)
 		if err != nil {
-			message.Fatal(err, err.Error())
+			return err
 		}
 		validator.DisplayFormattedMessage()
 		if !validator.IsSuccess() {
+			// TODO: Replace exit status with an error.
 			os.Exit(1)
 		}
```

Comment:
Noticed the TODO here. What do you think about replacing the exit status with an error?

Reply:
I think @AustinAbro321 has fixed this in another PR already.

Reply:
Yeah it's done here https://github.com/defenseunicorns/zarf/pull/2660